.github/workflows/test_terraform_webhook_module.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: Terraform module tests (webhook-gateway)
+
+on:
+  pull_request:
+    paths:
+      - "**/terraform/**"
+
+permissions:
+  contents: read
+
+jobs:
+  test-terraform:
+    name: Test Terraform with Juju
+    runs-on: ubuntu-latest
+    env:
+      WORKING_DIR: "webhook-gateway/charm/terraform/tests"
+    steps:
+      - uses: actions/checkout@v5.0.0
+      - uses: charmed-kubernetes/actions-operator@main
+        with:
+          provider: "microk8s"
+          channel: 1.33-strict/stable
```

Comment:
Should we try using the `1.34-strict/stable` which is the latest? (or would there be other reasons for using 1.33?)

Reply:
i think we can use 1.34-strict/stable, originally the template used 1.33-classic/stable so i used that as guidance and changed it to 1.33-strict/stable to be compatible with microk8s haha